### PR TITLE
build: Update all tools

### PR DESCRIPTION
**Pull request by jimmidyson**
**What problem does this PR solve?**:

Weekly tools update.

**Which issue(s) this PR fixes**:
Fixes #

**How Has This Been Tested?**:

**Special notes for your reviewer**:

### CODE REVIEW

File: hack/flakes/flake.nix [hunk 1]
```diff
@@ -14,16 +14,16 @@
...
             name = "go-mod-upgrade";
-            version = "0.9.1";
+            version = "0.10.0";
             src = fetchFromGitHub {
               owner = "oligot";
               repo = "go-mod-upgrade";
               rev = "v${version}";
-              hash = "sha256-+C0IMb7MU1fq/P0/tTUNmzznZ1q5M69491pO5yBZlVs=";
+              hash = "sha256-BuHyqv0rK1giNiPO+eCx13rJ9L6y2oCDdKW1sJXyFg4=";
             };
```
Comment (faiq) [inline hack/flakes/flake.nix:23]: can you tell us the command to upgrage this hash. i just use the wrong one until its right

**Review by faiq — APPROVED** [review]

File: hack/flakes/flake.nix (hunk 1 above)
Comment (jimmidyson) [reply hack/flakes/flake.nix:23]: 😂 that's what i do. I think there must be a command to calculate it, but what I end up doing is replacing this hash and vendor hash with `lib.fakeHash`, running the devbox install, and then updating the hashes after that. It's not pretty...

Comment (jimmidyson) [reply hack/flakes/flake.nix:23]: Here ya go:

```
$ nix hash to-sri --type sha256 $(nix-prefetch-url --unpack https://github.com/oligot/go-mod-upgrade/archive/v0.10.0.tar.gz 2>&/dev/null)

sha256-BuHyqv0rK1giNiPO+eCx13rJ9L6y2oCDdKW1sJXyFg4=
```

 Not sure how to do it for vendorHash though.

Comment (jimmidyson) [reply hack/flakes/flake.nix:23]: Official advice from https://ryantm.github.io/nixpkgs/languages-frameworks/go/#ex-buildGoModule:

> To obtain the actual hash, set vendorHash = lib.fakeSha256; and run the build

**Review by dlipovetsky — APPROVED** [review]

**Pull request merged**